include/mbedtls/config.h

```diff
@@ -2309,6 +2349,11 @@
  *
  * This module is required for SSL/TLS up to version 1.1, for TLS 1.2
  * depending on the handshake parameters, and for SHA1-signed certificates.
```

Comment:
Missing, by analogy with the MD5 module: for PBKDF2 when decrypting PEM-encoded encrypted keys.